server/server/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for server project.
+
+Generated by 'django-admin startproject' using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-)j0aj1v^(v+47-py7uws(4s&6(q!(mzc(i50e$oe+*w+#skbwg"
```

Comment:
we should probably hide this in an env var

Reply:
Good call. I honestly forgot this got generated for django.

Reply:
done, API_KEY added to environment variables.